src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
 
 const App = () => (
   <div>
-    <Navigation />
+    <Route exact path="/" component={Home}/>
+    <Route path="/contact" component={Contact}/>
```

Comment:
Try to use `exact` on all of them, play around with it, and see what it actually does. I usually use `exact` on all my paths, if i want my component to display on a separate page